### DISCUSSION TITLE
All imported Components are red even using volar-starter

### DISCUSSION THREAD

**Discussion by danielmoessner** [vuejs/language-tools#1079] [Q&A] [answered]
As you can see in the screenshot, the components WelcomeItem and DocumentationIcon are red. This happens in everyone of my projects. I want to replace vetur with volar. Vetur is already uninstalled and all other extensions are disabled. But still the components are red. Why is that? What might be happening?

I've used https://github.com/johnsoncodehk/volar-starter and changed nothing. I did restart VSCode already.

<img width="1079" alt="image" src="https://user-images.githubusercontent.com/38765196/159137712-19138e4a-75c2-4e5c-beab-54282771831c.png">

This only happens on <script setup lang="ts"> without setup it does not happen.

**Comment by xiaoxiangmoe** [comment]
Can you give me a reproduction demo?

**Reply by danielmoessner** [reply]
Well, it happens on the volar-starter: https://github.com/johnsoncodehk/volar-starter. I don't know if I've got something installed that breaks it, but I've got no global npm packages.

**Reply by danielmoessner** [reply]
Alle I can say is that i get the following error: load volar.config.js failed in ...path/volar-starter
I can see that message in the following VSCode outputs:
Volar - HTML
Volar - Document
Volar - API

**Comment by danielmoessner** [comment, marked as answer]
It is working now. I don't know why. Maybe because I've restarted the computer after Vetur was uninstalled.
